# Topic names
state_topic: "/ur3_cartesian_velocity_controller_sim/ee_state"
command_topic: "/ur3_cartesian_velocity_controller_sim/command_cart_vel"

# Static gain
Dx: 10
Dy: 10
Dz: 10
Kx: 50
Ky: 50
Kz: 50

